extensions/punto/src/Dict.ts

```diff
@@ -25,7 +25,7 @@ export const ru_en = new Map([
   ["№", "#"],
   ["%", "$"],
   [":", "%"],
-  [",", "^"],
+  [",", "б"],
```

Comment:
The former value was correct

Reply:
Yes, perhaps should add setting for a different keyboard.